ui/js/controllers/perf/alerts.js

```diff
@@ -529,6 +529,24 @@ perf.controller('AlertsCtrl', [
                 });
         };
 
+        $scope.summaryTitle = {
+            text: '<i class="fa fa-spinner" aria-hidden="true"></i>',
```

Comment:
I think `html` would be a better variable name here, since the content is not strictly "text".